repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -314,18 +330,18 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
       }
     }
 
-    def getEndpoints(getAdminToken: Boolean => Try[String], authToken: String, validToken: ValidToken): Try[EndpointsData] = {
+    def getEndpoints(authToken: String, validToken: ValidToken): Try[EndpointsData] = {
       logger.trace(s"Getting endpoints for: $authToken")
 
       // todo: extract the "make call with admin token and retry" logic since that pattern is used elsewhere
       Option(datastore.get(s"$ENDPOINTS_KEY_PREFIX$authToken").asInstanceOf[EndpointsData]) match {
         case Some(endpointsData) => Success(endpointsData)
         case None =>
-          getAdminToken(false) flatMap { adminToken =>
+          getValidatingToken(authToken, force = false) flatMap { adminToken =>
```

Comment:
Why use one named parameter and one not, but still in the method definition order?